### PR TITLE
build the feature branch

### PR DESCRIPTION
**Pull request by nicupop729**

### CODE REVIEW

File: src/class.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { inputTask } from './events.js';
+
+class Task {
+  constructor(description, completed = false, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+// eslint-disable-next-line
```
Comment (micheaol) [inline src/class.js]: Instead of disabling linters you can use window.location

Comment (micheaol) [inline src/class.js]: You can use window.location.reload() instead of disabling linter.

File: src/events.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+export const changeState = (value) => {
+  const checkBox = document.querySelectorAll('.checkbox');
+  _.forEach(checkBox, (check, i) => {
+    check.addEventListener('click', () => {
+      value[i].completed = check.checked;
+      saveTaskToLocal(value);
+      const itemDesc = Array.from(
+        document.querySelectorAll('.item-description'),
+      );
+      // eslint-disable-next-line
```
Comment (micheaol) [inline src/events.js]: Please ensure that all commented code are removed from your code-base, it makes your code cleaner.

File: src/class.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+export const addNewTask = () => {
+  const newTask = new Task(inputTask.value);
+  if (tasks.length === 0) newTask.index = 1;
+  if (tasks.length > 0) newTask.index = tasks.length + 1;
+  tasks.push(newTask);
+  saveTaskToLocal(tasks);
+  inputTask.value = '';
+  // eslint-disable-next-line
+  location.reload();
```
Comment (micheaol) [inline src/class.js]: You can use window.location.reload() instead of disabling linter.

**Review by micheaol — COMMENTED** [review]
Hi @nicupop729 ,

Great work @nicupop729 👍 

## To highlight:

    - Your code is clean and conform with Javascript best practice👏
    - Good PR description 👏
    
However, there are some optional suggestion:
  ### Optional suggestions
    -  Please check inline comments for more details.

Overall, great Job @nicupop729 ! and keep up the good work.

Happy coding!

File: src/class.js (hunk 1 above)
Comment (nicupop729) [reply src/class.js]: Thanks a lot for the suggestion!! Definitely, I will use this!!

File: src/events.js (hunk 2 above)
Comment (nicupop729) [reply src/events.js]: I will take care of that!

File: src/class.js (hunk 3 above)
Comment (nicupop729) [reply src/class.js]: cheers!!

**Pull request merged**